core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -330,6 +330,7 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     MustNotBeNextTo("Must not be next to [terrainFilter]", UniqueTarget.Building),
 
     Unsellable("Unsellable", UniqueTarget.Building),
+    ObsoleteWith("Obsolete with [tech]", UniqueTarget.Building, UniqueTarget.Resource),
```

Comment:
It looks like this can also apply to improvements (`tileInfo.kt:457`)? iirc maybe also units?

Reply:
Units already have an 'obsolete' field, and it's ubiquitous enough that I don't feel the need to convert it
Improvements...? Interesting

Reply:
OH I see what you mean
It was ALREADY for improvements 
I'll add that in then, and rewrite some of the common aspects